core/main/src/service/context_manager.rs

```diff
@@ -93,6 +94,14 @@ impl ContextManager {
 
         // Asynchronously get context and update the state
         tokio::spawn(async move {
+            // Set default cloud permissions value
+            ps_c.get_client().get_extn_client().context_update(
+                RippleContextUpdateRequest::UpdateFeatures(vec![FeatureUpdate::new(
+                    FEATURE_CLOUD_PERMISSIONS.into(),
+                    ps_c.get_device_manifest().get_features().cloud_permissions,
+                )]),
+            );
+
```

Comment:
Set default cloud_permissions feature when ContextManager initially starts up, may eventually be overridden by RFC flag.